frontend/src/features/videos/VideoCard.tsx

```diff
@@ -221,6 +223,11 @@ function VideoCard({
           {video.uploader && (
             <span className={classes.channel}>{video.uploader}</span>
           )}
+          {!!video.duration && video.duration && (
+            <span className={classes.youtube_complements_p}>
+              {shortEnglishHumanizer(video.duration * 1000)}
+            </span>
+          )}
```

Comment:
After some experiments, I end up with a potential solution to display the duration in the bottom-right corner of the player, using the [`wrapper` prop](https://github.com/cookpete/react-player#props).

Here is what the wrapper could look like:
```tsx
const PlayerWrapper = ({
  duration,
  children,
}: {
  duration?: string;
  children: React.ReactNode;
}) => {
  const [isDurationVisible, setIsDurationVisible] = useState(true);
  return (
    <Box
      position="relative"
      height="100%"
      onClick={() => setIsDurationVisible(false)}
    >
      {isDurationVisible && duration && (
        <Box
          position="absolute"
          bottom={0}
          right={0}
          bgcolor="rgba(0,0,0,0.5)"
          color="#fff"
          px={1}
          fontSize="0.8em"
          fontWeight="bold"
        >
          {duration}
        </Box>
      )}
      {children}
    </Box>
  );
};
```